### PR TITLE
feat(showcase): add airgap wallet

### PR DESCRIPTION
**Pull request by AndreasGassmann**
Added AirGap Wallet to the showcase

---

Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [ ] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [ ] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

**Review by yejidfinity — APPROVED** [review]

File: showcase.json [hunk 1]
```diff
@@ -1793,5 +1793,22 @@
     "usesInternetIdentity": true,
     "logo": "/img/showcase/bink_logo.png",
     "twitter": "https://twitter.com/BinkDapp"
+  },
+  {
+    "id": "AirGap",
```
Comment (sesi200) [inline showcase.json:1798]: somehow the ID doesn't accept upper case characters?!

File: showcase.json [hunk 2]
```diff
@@ -1793,5 +1793,22 @@
...
+    "name": "AirGap",
+    "oneLiner": "Protect your crypto and store your private keys offline by turning a spare phone into a Cold Wallet.",
+    "website": "https://airgap.it/",
+    "tags": ["Wallet"],
+    "description": "The AirGap solution turns a smartphone into a cold wallet using the AirGap Vault app. It handles your private key completely offline, whilst the designated companion – the AirGap Wallet, gives you a portfolio overview and initiates and broadcasts transactions for the Vault to sign offline.\nTransactions are signed seamlessly without the use of cables, thanks to verifiable QR-codes.\nTo ensure top security, AirGap has a unique approach to creating a secure key by utilizing various sources of entropy such as video, sound, gyroscope, and touch. This ensures maximum randomness and therefore results in a secure key generation.",
```
Comment (sesi200) [inline showcase.json:1803]:
```suggestion
    "description": "The AirGap solution turns a smartphone into a cold wallet using the AirGap Vault app. It handles your private key completely offline, whilst the designated companion – the AirGap Wallet – gives you a portfolio overview and initiates and broadcasts transactions for the Vault to sign offline.\nTransactions are signed seamlessly without the use of cables, thanks to verifiable QR-codes.\nTo ensure top security, AirGap has a unique approach to creating a secure key by utilizing various sources of entropy such as video, sound, gyroscope, and touch. This ensures maximum randomness and therefore results in a secure key generation.",
```
This whole section is very long, and looks a bit weird on the page. Do you think you could shorten it a bit?

**Comment by sesi200** [comment]
If you want me to be able to merge master you have to give me write-access to the branch. This is easiest with the button on the right that gives maintainers access to your branch

**Comment by AndreasGassmann** [comment]
Hi @sesi200. Thanks for the feedback. I tried to give you access to the branch, but I somehow can't find that option. I remember seeing it when I did other PRs, maybe it's something about the permissions.

I invited you to the project, hopefully that works too!

**Comment by domwoe** [comment]
Since AirGap is now already part of the ecosystem page, I think we can close this PR. Correct @AndreasGassmann ?

**Pull request closed without merging**